webapp/src/components/user_popover/badge_list.tsx

```diff
@@ -109,7 +111,7 @@ class BadgeList extends React.PureComponent<Props, State> {
         let row: React.ReactNode[] = [];
         for (let i = 0; i < toShow; i++) {
             const badge = this.state.badges![i];
-            if (i !== 0 && i % BADGES_PER_ROW === 0) {
+            if (i !== 0) {
```

Comment:
The logic changed here from "if this is not the first badge, and we are at the beginning of a row" to just "this is not the first badge"

What are the implications of this?

Reply:
Since we are only showing one row, there is no implication. However this lets us to further simplify the logic. Take a look at my last commit